api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/model/support/dynamodb/AbstractDynamoDbProperties.java

```diff
@@ -53,4 +69,12 @@
      */
     private boolean localInstance;
 
+    /**
+     * Mirrors the <a href="https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/dynamodbv2/model/BillingMode.html">BillingMode</a> enum in the AWS client.
+     *
+     */
+    public enum BillingMode {
+        PROVISIONED,
```

Comment:
For each enum field, describe in sufficient detail what each field does. Avoid referencing external links and rely on descriptions to assist with configuration metadata assembly and docs.